apps/web/app/globals.css

```
@@ -374,18 +374,26 @@ pre {
-  border-radius: 999px;
+  border-radius: 16px;
   border: 1px solid var(--line-strong);
   background: rgba(225, 215, 255, 0.03);
   color: var(--ink-soft);
   font-family: var(--font-mono), monospace;
   font-size: 0.8rem;
   max-width: 100%;
-  overflow-x: auto;
-  white-space: nowrap;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)